google-cloud-bigquery-data_transfer/lib/google/cloud/bigquery/data_transfer/v1/doc/google/cloud/bigquery/datatransfer/v1/datatransfer.rb

```diff
@@ -0,0 +1,500 @@
+# Copyright 2018 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     https://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+module Google
+  module Cloud
+    module Bigquery
+      module Datatransfer
```

Comment:
Hm I think the two modules is expected since the `Datatransfer` module is generated by protoc which we have no control over. We _could_ have the module be `Datatransfer` for the generated client for consistency.

Reply:
Is this the same problem as https://github.com/GoogleCloudPlatform/google-cloud-ruby/issues/1762 and https://github.com/googleapis/artman/issues/288?

Reply:
@jbolinger Yes.

Reply:
I am simply going to make hand-edits to the protobuf files based on what the outcome of resolving https://github.com/googleapis/artman/issues/288 would be.

Reply:
So the module in this `doc` file will be updated to `DataTransfer` with path `data_transfer`, like the production files?